### ISSUE TITLE
Error while adding custom language

### ISSUE THREAD

**Issue by br4ch1st0chr0n3** [mfornet/acmx#141]
Hi! I'm trying to add V language support, but get RTE on Test 0 on Ubuntu 20.04.

Here's what I did so far. 
1. Installed V ([link](https://github.com/vlang/v#installing-v---from-source-preferred-method))
2. Symlinked compiler ([link](https://github.com/vlang/v#symlinking))
3. Got a folder with tests and `sol.v` via Competitive Companion and acmx
4. Pasted the solution to Codeforces problem [1666L](https://codeforces.com/problemset/problem/1666/L) into `sol.v`
5. Here is it ([link](https://gist.github.com/br4ch1st0chr0n3/758fdaeb43ff79247c422ac16dde267b))
6. Tried running in terminal with `v run sol.v < testcases/0.in` and got an expected answer
7. Created an executable with `v sol.v`, run `./sol < testcases/0.in` and got an expected answer
8. Created a custom `v.json` for running in `.acmx/languages`.
```json
{
    "preRun": ["v $CODE; cp $CODE $OUTPUT"],
    "run": ["$OUTPUT"],
    "ext": "v"
}
```
9. Opened command palette in VS Code, selected `ACMX: Run`
10. Got RTE on Test 0
11. With `.cpp` files, I get just WA.

Please, correct my steps.

**Comment by mfornet** [comment]
First of all, thanks for bringing v-lang to my attention :)

tl;dr: Use the following configuration:

```
{
    "preRun": [
        "v",
        "$CODE"
    ],
    "run": [
        "$ATTIC/../sol"
    ],
    "ext": "v"
}
```

There are few problems with your configurations, in `preRun`. 

1. It doesn't actually support calling multiple methods the way you are doing (compile + copy separated by `;`). You are supposed to each element of the command as a single element of the array so it is properly shell encoded.
2. The second time `$CODE` is substituted; it is done with the wrong value; you probably expect `sol` while you will get the file with the extension `sol.v`

The cleaner way would be to pass the output file to the compiler; but as far as I can see this is not supported by `v` CLI. In g++ you would use `g++ main.cpp -o attic/my_solution.out`. Unfortunately there is no clean way to refer to the compiled binary from `acmx`.

If you are ok with it; you can assume your main file will always be called `sol.v` so you can assume after compiling with `v sol.v` the output will be `sol`. And you can refer to this file using the provided placeholder `$ATTIC` and go back one folder like: `$ATTIC/../sol`.

Check above a hacky but working version of v-lang. I'll be closing the issue, but feel free to ask any question if it doesn't work for you.

**Comment by br4ch1st0chr0n3** [comment]
Hi! Thanks. But I tried another way. Simply wrote command terms in separate strings. Here's how `vlang.json` looks like ([link](https://github.com/br4ch1st0chr0n3/vforces/blob/main/vlang.json)).
